evaluation_scripts/plot_functions.py

```diff
@@ -0,0 +1,296 @@
+# This script contains the functions used for plotting different plots in
+# other scripts used for evaluation.
+
+# Author: Shweta Narkhede and Camilo Salcedo
+# Created on: Oct 24th, 2020
+# %%
+import pandas as pd
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import eval_functions as ef
+import seaborn as sns
+
+#%% Functions
+
+def get_histogram(forecasts1, obs_week, title_string):
+    """Get Histograms:
+    -----------------------------------
+    This function plots histograms of predicted weekly flow data and
+    the count of the flow value prediction. It also plots observed weekly
+    data for last week for comaparision.
+    -----------------------------------
+    Parameters:
+    forecasts1  = array
+                every student's forecast for week 1
```

Comment:
couldn't this be fore week1 or week2?

Reply:
Thanks for your comment. The parameter "forecasts1" was generalized as "forecasts" because it could be either for Week1 or Week 2.